components/scream/src/share/io/scorpio_output.hpp

```diff
@@ -27,7 +27,12 @@
  *  Casename:                     STRING
  *  Averaging Type:               STRING
  *  Max Snapshots Per File:       INT                   (default: 1)
- *  Fields:                       ARRAY OF STRINGS
+ *  Grids:                        ARRAY OF STRINGS
```

Comment:
I'm a bit confused here, from the code above I understood that one can also just list "Fields" at the level of "Grids" like is currently done.  Maybe I misunderstood the checks though.

Reply:
I will go over this documentation, and make sure it's consistent.

Reply:
Ah, yeah, it is possible, but the AD (or, better, the OutputManager) was not really supporting it, making it effectively pointless. I'll add this possibility back.